### PR TITLE
Fix accessing contentDocument on unbind when it is null

### LINKED ISSUE #13: this.contentDocument is null for safari mobile safari
I am having this error for safari/mobile safari 

```
TypeError: null is not an object (evaluating 'this.contentDocument.defaultView')
```
I will keep debugging

### PR DESCRIPTION
**Pull request by tirli**
Fixes #13 
Please merge and publish to NPM as this bug blocks not only me (according to issue)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,11 +77,13 @@ module.exports.unbind = function (element, fn) {
     if (attachEvent) {
       element.detachEvent('onresize', resizeListener)
     } else if (element.__resizeTrigger__) {
-      element.__resizeTrigger__.contentDocument.defaultView.removeEventListener(
-        'resize',
-        resizeListener
-      )
-      delete element.__resizeTrigger__.contentDocument.defaultView.__resizeTrigger__
+      if (element.__resizeTrigger__.contentDocument && element.__resizeTrigger__.contentDocument.defaultView) {
```
Comment (eaglus) [inline index.js]: Please, move element.__resizeTrigger__.contentDocument to variable for better readability, and i will merge it.

Comment (eaglus) [reply index.js]:
```
 var contentDocument = element.__resizeTrigger__.contentDocument;
 var defaultView = contentDocument && contentDocument.defaultView;
 if (defaultView) {
   defaultView.removeEventListener('resize', resizeListener);
   delete defaultView.__resizeTrigger__;
 }
```

**Review by eaglus — CHANGES_REQUESTED** [review]

**Comment by tirli** [comment]
@eaglus updated

**Comment by tirli** [comment]
@eaglus Thank you but seems like it's still not published. Can you do it or only @KyleAMathews have an access?

**Comment by eaglus** [comment]
> @eaglus Thank you but seems like it's still not published. Can you do it or only @KyleAMathews have an access?

Hi Tirli! I can't publish this. I wrote email to KyleAMathews, to give me right for publishing, now i'm waiting for response

**Pull request merged**